pkg/Microsoft.NETFramework.Compatibility/Microsoft.NETFramework.Compatibility.pkgproj

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <PackageVersion>2.1.0</PackageVersion>
+  </PropertyGroup>
+
+  <ItemDefinitionGroup>
+    <LibraryPackage>
+      <Version>4.5.0</Version>
+    </LibraryPackage>
+  </ItemDefinitionGroup>
+
+  <ItemGroup>
+    <LibraryPackage Include="System.CodeDom" />
```

Comment:
I was trying to come up with a way to validate not only that it is complete but also things had matching versions but I couldn't come up with anything clever. Let me know if you have any ideas.